parachain/src/command.rs

```diff
@@ -458,6 +458,28 @@ pub fn run() -> Result<()> {
 
             Ok(())
         }
+        #[cfg(feature = "try-runtime")]
+		Some(Subcommand::TryRuntime(cmd)) => {
+			let runner = cli.create_runner(cmd)?;
+            let chain_spec = &runner.config().chain_spec;
+
+            with_runtime_or_err!(chain_spec, {
+                return runner.async_run(|config| {
+                    // we don't need any of the components of new_partial, just a runtime, or a task
+                    // manager to do `async_run`.
+                    let registry = config.prometheus_config.as_ref().map(|cfg| &cfg.registry);
+                    let task_manager =
+                        sc_service::TaskManager::new(config.tokio_handle.clone(), registry)
+                            .map_err(|e| sc_cli::Error::Service(sc_service::Error::Prometheus(e)))?;
+                    Ok((cmd.run::<Block, Executor>(config), task_manager))
+                });
+            })
+			
+		},
+		#[cfg(not(feature = "try-runtime"))]
+		Some(Subcommand::TryRuntime) => Err("TryRuntime wasn't enabled when building the node. \
```

Comment:
Why not simply exclude this if the feature isn't enabled?

Reply:
So that if you try to use the command without the proper features, you get a more helpful error message. This was copied from the reference, and I didn't see a need to change that